velox/benchmarks/tpch/CMakeLists.txt

```diff
@@ -35,7 +36,9 @@ target_link_libraries(
   velox_vector_test_lib
   Folly::folly
   ${FOLLY_BENCHMARK}
-  fmt::fmt)
+  fmt::fmt
+  ${AWSSDK_LIBRARIES}
```

Comment:
Please use targets and avoid variables as it is not modern cmake.

Reply:
I don't know the target for aws sdk. I copied the dependency from the S3 connector CMakeList.txt.

Reply:
Ok, I just discovered that the variable is not required. Only the curl reference is required.